### ISSUE TITLE
PlatformAttribute has AllowMultiple = true, but that doesn't work.

### ISSUE THREAD

**Issue by manfred-brands** [nunit/nunit#3941]
Found whilst working on #3926 

On a test with more than one attribute:

```
[Platform(Include = "Linux")]
[Platform(Include = "Win")]
```
I do get the error: _Only supported on Linux_ when running under Windows.

Yes, the documentation mentions combining the platforms using commas, but should we make this a compile error?

Should the `AttributeUsage` be update to `AllowMultiple = false`?

**Comment by rprouse** [comment]
One of the problems with our platform attribute is that it is used for multiple things, so we could have 

```csharp
[Platform(Include = "Linux")]
[Platform(Include = "64-Bit")]
```

**Comment by stevenaw** [comment]
If multiple is desired, then I think @manfred-brands fixed it in main via https://github.com/nunit/nunit/pull/3926
Should the relevant parts of that PR be backported to 3.13.x in order to close this?

**Comment by stevenaw** [comment]
I am going to close this as fixed in 4.0 as it was done in #3926 based on this comment: https://github.com/nunit/nunit/pull/3926#discussion_r716139116

It doesn't look like we want to backport it but any future effort there is to be tracked under a separate issue.

I'll put up a PR against the docs repo shortly to ensure this is in the 4.0 release notes as well
